.github/workflows/mvn-verify-check.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         java-version: [8, 11]
-        spark-version: ['313', '324', '334', '350']
+        spark-version: ['324', '334', '350', '400']
```

Comment:
Is this working? I thought that we cannot use spark-4.0.0 yet as it is based on scala 2.13.

- the maven repository has spark-4.0.0-previews only  https://mvnrepository.com/artifact/org.apache.spark/spark-core
- the archive links also do not have scala2.12 binaries in the spark-4.0.0. https://archive.apache.org/dist/spark/spark-4.0.0-preview2/

Reply:
We added support Spark 4.0.0 in #537. This change adds it to github workflow. All tests are passing on Spark 4.0.0.

Reply:
Yes, you are right. We added it long time ago.
Then we removed it from workflow when the scala2.12 was dropped.
I believe this success is bogu because it fetches an outdated snapshot of Spark4.0.0 that was compatibe with scala2.12. It is dated september 2023
https://repository.apache.org/content/repositories/snapshots/org/apache/spark/spark-core_2.12/

But if you look closely to scala2.13, you will see that up-to-date artifacts are January 2025
https://repository.apache.org/content/repositories/snapshots/org/apache/spark/spark-sql_2.13/

We should remove that `400` from the github-workflow because it is not really testing Spark-400

Reply:
Ahh. I see. Thank you for this catch. I have removed it from the workflow.